### PR TITLE
issue-migration

### PR DESCRIPTION
**Pull request by austininseoul**

### CODE REVIEW

File: frontend/layouts/default.vue [hunk 1]
```diff
@@ -2,9 +2,8 @@
   <v-app>
     <v-navigation-drawer
       v-model="drawer"
-      style="height: 99vh;"
```
Comment (jmformenti) [inline frontend/layouts/default.vue:5]: why are you removing this?

File: frontend/layouts/default.vue [hunk 2]
```diff
@@ -2,9 +2,8 @@
...
       color="primary"
-      :permanent="!$vuetify.breakpoint.sm && !$vuetify.breakpoint.xs && !$vuetify.breakpoint.md"
```
Comment (jmformenti) [inline frontend/layouts/default.vue:7]: why are you removing this and added fixed?

File: frontend/layouts/default.vue [hunk 3]
```diff
@@ -127,13 +179,30 @@
...
       </v-tooltip>
     </v-app-bar>
-    <v-main class="min-height-full-display">
+    <v-main>
       <v-container fluid ma-50>
         <v-breadcrumbs :items="$store.state.breadcrumb.items" />
         <nuxt />
       </v-container>
     </v-main>
-    <philo-footer />
```
Comment (jmformenti) [inline frontend/layouts/default.vue:136]: why are you removing the footer? we are losing some options

File: frontend/layouts/default.vue (hunk 1 above)
Comment (jmformenti) [inline frontend/layouts/default.vue:5]: why are you removing this?

File: frontend/layouts/default.vue (hunk 2 above)
Comment (jmformenti) [inline frontend/layouts/default.vue:7]: why are you removing this and adding "fixed"?

File: frontend/layouts/default.vue [hunk 4]
```diff
@@ -81,20 +80,73 @@
...
       dark
+      src="/img/header_page-opacity15_2.gif"
     >
       <template #img="{ props }">
         <v-img
           v-bind="props"
           gradient="to top right, rgba(33, 33, 33,.7), rgba(250, 250, 250,.7)"
         />
       </template>
-      <v-app-bar-nav-icon class="d-lg-none" @click.stop="drawer = !drawer" />
```
Comment (jmformenti) [inline frontend/layouts/default.vue:97]: why are you removing this?

File: frontend/layouts/default.vue [hunk 5]
```diff
@@ -127,13 +179,30 @@
         <span>{{ $t('auth.login.label') }}</span>
       </v-tooltip>
     </v-app-bar>
-    <v-main class="min-height-full-display">
```
Comment (jmformenti) [inline frontend/layouts/default.vue:130]: why are you removing this?

File: frontend/pages/wiki/_page.vue [hunk 6]
```diff
@@ -1,6 +1,5 @@
 <template>
   <v-container>
-    <search-simple />
```
Comment (jmformenti) [inline frontend/pages/wiki/_page.vue:3]: why are you removing the search box?

File: frontend/pages/wiki/_page.vue [hunk 7]
```diff
@@ -17,7 +16,7 @@ export default {
   },
 
   created () {
-    this.pageName = this.removeLocalePrefix(this.$route.params.page)
```
Comment (jmformenti) [inline frontend/pages/wiki/_page.vue:20]: I think that all the changes from here to the end are related to try to fix broken links
If so, that was fixed in this commit: https://github.com/PhiloBiblon/philobiblon-ui/commit/a761338ca3fe2bbf7b05b773407877d361278f37
Let's discuss in our next meeting

**Review by jmformenti — CHANGES_REQUESTED** [review]
Additionally you need to review all the lang/*.js files, we are losing labels and adding deprecated ones, I think is a mix with a previous version

File: frontend/layouts/default.vue (hunk 3 above)
Comment (austininseoul) [reply frontend/layouts/default.vue:136]: let me review, there's some version difference i think here

**Pull request closed without merging**